### PR TITLE
[COH-17363] Enhance/control renoir caches

### PR DESCRIPTION
**Pull request by palikar**

### CODE REVIEW

File: front_end/core/sdk/CohtmlDebugModel.ts [hunk 1]
```diff
@@ -100,9 +100,24 @@ export class CohtmlDebugModel extends SDKModel<void> implements ProtocolProxyApi
...
+
+  async getRenoirCahcesState() : Promise<Protocol.CohtmlDebug.RenoirCachesState|null> {
+    const response = await this.cohtmlDebugAgent.invoke_getRenoirCachesState();
+    if (response.getError()) {
+      return null;
+    }
+    return response.stats;
+  }
+
+  setRenoirCacheState(stateToSet: Protocol.CohtmlDebug.RenoirCache) {
```
Comment (alien1976) [inline front_end/core/sdk/CohtmlDebugModel.ts]:
```suggestion
  setRenoirCacheState(stateToSet: Protocol.CohtmlDebug.RenoirCache): void {
```

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts [hunk 2]
```diff
@@ -204,6 +236,141 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
...
+    let newCacheUIEntry = new RenoirCacheUIEntry();
+
+    let firstRow = cachePanel.createChild('div', '');
+    {
+      let entryName = firstRow.createChild('span', 'info-span');
+      entryName.innerText = UIStrings.capacityStr;
+
+      // input field where the user enters the desired cache size
+      let entryInput = firstRow.createChild('input');
+      entryInput.setAttribute('type', 'text');
```
Comment (alien1976) [inline front_end/entrypoints/inspector_main/CohtmlPanel.ts]: Why the input type is not a number?

File: third_party/blink/public/devtools_protocol/browser_protocol.json [hunk 3]
```diff
@@ -20809,6 +20809,50 @@
...
                         }
                     ]
+                },
+                {
+                    "id": "RenoirCache",
+                    "type": "object",
+                    "properties": [
+                        {
+                            "name": "type",
+                            "$ref": "strign"
```
Comment (alien1976) [inline third_party/blink/public/devtools_protocol/browser_protocol.json:20819]:
```suggestion
                            "$ref": "string"
```

**Review by alien1976 — CHANGES_REQUESTED** [review]
* Maybe add max/min value to the input fields because it assets if > 3gb
image
* Used images section is broken (the green color highlights the locations where it should be shown)
image
* When you resize the inspector the input fields are overflowing and are not responsive. Either fix this or create a task in the FE board so we can fix it when we have time.
image

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts (hunk 2 above)
Comment (palikar) [reply front_end/entrypoints/inspector_main/CohtmlPanel.ts]: sure, fixed

File: front_end/core/sdk/CohtmlDebugModel.ts (hunk 1 above)
Comment (palikar) [reply front_end/core/sdk/CohtmlDebugModel.ts]: fixed

File: third_party/blink/public/devtools_protocol/browser_protocol.json (hunk 3 above)
Comment (palikar) [reply third_party/blink/public/devtools_protocol/browser_protocol.json:20819]: fixed

**Review by omalinov — APPROVED** [review]

**Review by alien1976 — APPROVED** [review]

**Pull request merged**